cypress/integration/change-annotation.spec.js

```diff
@@ -29,7 +29,7 @@ describe('Annotation Change Test in Project documents view and User My annotatio
             cy.wrap(container).contains("Submit").click()
 
             // Default annotation should be the latest
-            cy.wrap(container).contains('"sentiment": "neutral"')
```

Comment:
It doesn't look like an issue, but are changes to this file meant to be in this PR?

Reply:
Yes, one of the updated packages changed the JSON output formatting so I'd updated the test case to match it.